recipes/openml/meta.yaml

```
@@ -0,0 +1,68 @@
+{% set name = "openml" %}
+{% set version = "0.11.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/{{ name }}/{{ name }}-python/archive/v{{ version }}.tar.gz
+  sha256: 360461d27c7795ee7d6a86cd15e0108b2876d10a4120afc8417e9ce64d0cb026
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - liac-arff >=2.4.0
+    - numpy >=1.6.2
+    - pandas >=1.0.0
+    - python >=3.6
+    - python-dateutil
+    - requests
+    - scikit-learn >=0.18
+    - scipy >=0.13.3
+    - xmltodict
+
+test:
+  imports:
+    - openml
+    - openml.datasets
+  commands:
+    - pip check
+    - pytest --pyargs {{ name }}
+  requires:
+    - flaky
+    - jupyter_client
+    - matplotlib
+    - mypy
+    - nbconvert
+    - nbformat
+    - oslo.concurrency
+    - pre-commit
+    - pip
+    - pyarrow
+    - pytest
+    - pytest-cov
+    - pytest-timeout
+    - pytest-xdist
+  source_files:
+    - tests
```

It's probably better not to run the unit tests of they have this many optional dependencies as it will make it very hard to notice if the `run` dependencies are enough on their own.
```suggestion
  requires:
    - pip
```

---

Is there any other way to run them or should I just skip them then? I initially just used an import test which was successful.

---

On a general note, is there a way to fix the `File name too long` error? I just haven't encountered it before so I'm just curious.

---

It's a bug in the packages tests:

https://github.com/openml/openml-python/blob/ab793a65efe42da18264252eceec4085f3e68b9f/tests/conftest.py#L40-L47

---

Makes sense, thank you!